test/integration/f22/Makefile

```diff
@@ -36,4 +36,4 @@ remote_sync: remote_setup
 	ssh $(SSH_USER)@$(SSH_HOST) "cd kpatch-test/kpatch && make"
 
 remote_slow: remote_sync
-	ssh $(SSH_USER)@$(SSH_HOST) "cd kpatch-test/kpatch/test/integration/f22 && make slow"
+	ssh $(SSH_USER)@$(SSH_HOST) "cd kpatch-test/kpatch/test/integration/f22 && make local"
```

Comment:
Because of
```
all:
 	$(error please specify local or remote)
```
So my intention is that the main user visible targets are "local" and "remote".  Where "local" defaults to "slow" and "remote" defaults to "remote_slow".  The other targets like "slow", "quick", and "cached" are hidden for more advanced users.